entrypoint.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
-set -e
```

Comment:
I believe you can leave `set -e` here. https://serverfault.com/questions/143445/what-does-set-e-do-and-why-might-it-be-considered-dangerous